docs/guides/api/api.primary.in.rst

```diff
@@ -0,0 +1,53 @@
+The Application Programming Interface (API) allows users to interact with Rubin's data services remotely.
+An API (Application Programming Interface) is a structured way for software applications to communicate with each other.
+It can be thought of as a back-and-forth exchange between a client and a server: one program (the client) makes a request for data, and another program (the server) provides a response.
+
+In the context of the RSP, the API aspect allows access to Rubin data—including images and catalogs—both within the RSP environment (e.g., via the RSP JupyterLab Notebook aspect) and externally (e.g., through the Tool for OPerations on Catalogues And Tables (`TOPCAT <https://www.star.bris.ac.uk/~mbt/topcat/>`_) or a `PyVO <https://pyvo.readthedocs.io/en/latest>`_-enabled environment like the National Optical-Infrared Astronomy Research Laboratory’s `Astro Data Lab <https://datalab.noirlab.edu/>`_).
+
+**Use the API Aspect to:**
+
+- Access Rubin data from outside the RSP.
+- Automate data queries or downloads.
+- Integrate Rubin data access into your own software tools.
+- Access to external datasets in addition to Rubin data.
+
+API Services
+============
+
+The API aspect of the RSP supports several types of services.
```

Comment:
Can you add something like, `primarily based on IVOA protocols.`? (linking to www.ivoa.net)